woocommerce/woocommerce-gutenberg-products-block
# Icons: add star and radio button icons, organize folder

This PR updates the icon components, pulling each into their own folder. I've also added the radio selected & unselected icons, and the star icon (used for Featured Product). Additionally, we've switched to using the Material checkbox icons directly, rather than a masked SVG (just to save some rendering). This matches the radio icons, but might appear slightly larger now – if so, will fix in CSS in a following PR.

### Screenshots

New checkmarks:

![screen shot 2019-01-09 at 12 15 38 pm](https://user-images.githubusercontent.com/541093/50916804-10776780-140a-11e9-8418-88ace6b4fd9e.png)

Icons for blocks:

![screen shot 2019-01-09 at 12 29 10 pm](https://user-images.githubusercontent.com/541093/50916854-33a21700-140a-11e9-85d3-33ac53ce64ad.png)


### How to test the changes in this Pull Request:

1. Look at all block icons, there should be no change
2. Add a Products by Category or Hand-picked Products block, the selection UIs should have working checkmark icons.
